lib/src/browser/disposable_browser.dart

```diff
@@ -212,28 +208,13 @@ class Disposable implements disposable_common.Disposable {
           onError: onError, onDone: onDone, cancelOnError: cancelOnError);
 
   @override
-  disposable_common.Disposable manageAndReturnDisposable(
-          disposable_common.Disposable disposable) =>
-      _disposable.manageAndReturnDisposable(disposable);
-
-  @override
-  T manageAndReturnTypedDisposable<T extends disposable_common.Disposable>(
-          T disposable) =>
-      _disposable.manageAndReturnTypedDisposable(disposable);
+  T manageDisposable<T extends disposable_common.Disposable>(T disposable) =>
```

Comment:
We might as well just name this more simply since we now have the version we always wanted :-)